docs/design/kademlia-removal.md

```diff
@@ -0,0 +1,86 @@
+# Kademlia Removal
+
+## Abstract
+
+This design document outlines the communication protocol between satellites and 
+storage nodes, network refreshes, and kademlia removal given the satellite opt-in 
+capability for storage nodes. 
+
+## Background
+
+User research on how Storage Node Operators would connect to satellites resulted in a 
+simple solution called Opt-in SNO select. Storage Nodes select which Satellites to work 
+with, hence the Storage Nodes do not need to discover Satellites on the network and 
+Satellites have no benefit from finding Storage Nodes on the network.
+
+At the time of writing, a storage node operator can input a list of trusted satellite 
+IDs and addresses into their configuration file on setup. There are several tardigrade-level 
+satellites already listed as a preset. This list is imported into a Go map data structure 
+within the “trust” package. Future improvements will enable users to manage 
+this list through a web console. Due to the nature of this change, the Storj network 
+at this point in time no longer requires the Kademlia DHT or related entities. 
+
+We will replace this protocol with direct communication between satellites and 
+storage nodes, and keep the network fresh without kademlia node discovery and random 
+lookups.
+
+## Design
+
+### Storage Nodes reach out to satellites that they want to work with
+    - Eventually during the design and implementation of trusted satellite management, 
+    this configured list may be imported into a on-disk data store, eg SQL table. Users will 
+    be able to modify this list through the storage node operator console. However, this is a 
+    separate discussion from this design document. This design is satisfied with a static trusted 
```

Comment:
By "static" here, I assume you mean it must be edited manually?

Reply:
Yes, I can clarify